FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -71,6 +64,14 @@ protocol SessionsProvider {
         installations: installations
       )
     )
+    let coordinator = SessionCoordinator(
+      identifiers: identifiers,
+      installations: installations,
+      fireLogger: fireLogger,
+      sampler: SessionSampler(sessionSamplingRate: settings.samplingRate),
+      loggingEnabled: settings.sessionsEnabled
```

Comment:
I think that depends on the behavior we want. Currently, by passing in individual primitive values in constructor, configs are only changed at the start of an app. Alternatively, we can pass in `Settings` and query the value when using it, which can lead to the values changing every session.

Reply:
Letting @samedson to weigh-in on this. My recommendation would be to apply the most recent configs values downloaded rather than wait until the next cold start.